build_debian.sh

```diff
@@ -315,7 +315,7 @@ sudo tee -a $FILESYSTEM_ROOT/etc/network/interfaces > /dev/null <<EOF
 
 auto eth0
 allow-hotplug eth0
-iface eth0 inet dhcp
+iface eth0 inet static
```

Comment:
I thought we are not going to disable dhcp globally? the dhcp is disabled only from ftos to sonic fast-boot migration?

Reply:
That was the earlier thought. But I could not disable this during migration since squashfs is readonly... So, changed the approach. Note that with this change, though we disable dhcp globally during the build, if there is no IP configured in the minigraph, the interfaces-config.sh would override the startup /etc/network/interfaces with a config that is set to dhcp.